cli/remotebazel/remotebazel_test.go

```diff
@@ -286,6 +288,11 @@ func TestGitConfig_BranchAndSha(t *testing.T) {
 		if len(tc.expectedPatches) > 0 {
 			require.Contains(t, string(config.Patches[0]), tc.expectedPatches[0])
 		}
+
+		// Clean up remote repo for future test cases
+		if updatedRemote {
+			testshell.Run(t, remoteRepoPath, "git checkout master && git reset HEAD~ && git checkout .")
+		}
```

Comment:
nit: might be a little clearer/more robust to run something like `git clean -fdx && git checkout <starting_commit_sha>`, which resets the current repo to a pristine state (no local diffs, untracked files etc.) then checks out the SHA that we're expecting for the next loop iteration

Reply:
Good call - simplified this